upgrade/upgrade-scripts/execute

```diff
@@ -68,7 +68,7 @@ EOF
 
 apt_get update || die "failed to update apt sources"
 
-INSTALLED_VERSION=$(dpkg-query -Wf '${Version}' delphix-entire)
+INSTALLED_VERSION=$(dpkg-query -Wf '${Version}' delphix-entire-generic)
```

Comment:
I think this code is outdated. There is no `delphix-entire-generic` package anymore.
We should also check for currently installed flavor of `delphix-entire-PLATFORM` and then install the same flavor.
We can use: 
```
dpkg-query -Wf '${Package}' 'delphix-entire-*'
dpkg-query -Wf '${Version}' 'delphix-entire-*'
```